mscore/inspector/inspector.cpp

```diff
@@ -391,8 +391,13 @@ void InspectorElementBase::setElement()
 
 void InspectorElementBase::autoplaceChanged(bool val)
       {
-      for (auto i : std::vector<QWidget*> { e.offsetX, e.offsetY, e.resetX, e.resetY, e.hRaster, e.vRaster })
+      for (auto i : std::vector<QWidget*> { e.offsetX, e.offsetY, e.resetX, e.resetY, e.hRaster, e.vRaster }) {
             i->setEnabled(!val);
+            if (val) {
+                  e.offsetX->setValue(0.0);
```

Comment:
Actually this just fixes a display problem in inspector, as after deselect/select these offsets are 0 again anyway.